src/altinn/tjenester.ts

```diff
@@ -0,0 +1,181 @@
+import { gittMiljo } from '../utils/environment';
+import * as Record from '../utils/Record';
+
+export type AltinnskjemaId =
+    | 'mentortilskudd'
+    | 'inkluderingstilskudd'
+    | 'ekspertbistand'
+    | 'lønnstilskudd'
+    | 'inntektsmelding';
+
+export type AltinntjenesteId =
+    | 'arbeidstrening'
+    | 'arbeidsforhold'
+    | 'midlertidigLønnstilskudd'
+    | 'varigLønnstilskudd'
+    | 'iaweb'
+    | 'pam';
+
+
+export interface AltinnFellesInfo {
+    navn: string;
+    tjenestekode: string;
+    tjenesteversjon: string;
+    beOmTilgangBeskrivelse: string; /* Fravær av beskrivelse betyr man ikke kan søke om tilgang */
+}
+
+export interface Altinnskjema extends AltinnFellesInfo {
+    sort: 'skjema';
+    skjemaUrl: string;
+}
+
+export interface Altinntjeneste extends AltinnFellesInfo {
+    sort: 'tjeneste';
```

Comment:
kanskje bedre å bruke type i stedet for sort? kan jo bety sortere på engelsk. men er ikke så farlig